scripts/ci/in_container/check_environment.sh

```diff
@@ -217,15 +220,21 @@ check_integration rabbitmq "nc -zvv rabbitmq 5672" 20
 check_integration cassandra "nc -zvv cassandra 9042" 20
 check_integration openldap "nc -zvv openldap 389" 20
 
-resetdb
-
 if [[ ${EXIT_CODE} != 0 ]]; then
     echo
     echo "Error: some of the CI environment failed to initialize!"
     echo
     exit ${EXIT_CODE}
 fi
 
+# Try to reset the database regardless of DB_EXIT_CODE. If this fails, the whole script will fail
+resetdb
```

Comment:
Are we in `set -e` mode here, or would we not actually know if this failed?

Reply:
Yep. we we are in -e mode.